src/main/scala/com/hacklanta/formality/Formality.scala

```diff
@@ -12,10 +12,17 @@ object Formality extends FieldValueHelpers {
   def field[T](selector: String, initialValue: T)(implicit valueConverter: (String)=>Box[T], valueSerializer: (T)=>String): SimpleFieldHolder[T, T, T] = {
     SimpleFieldHolder(selector, Full(initialValue), Nil, Nil, Nil)(valueConverter, valueSerializer)
   }
+  def field[T](selector: String, initialValue: Box[T])(implicit valueConverter: (String)=>Box[T], valueSerializer: (T)=>String): SimpleFieldHolder[T, T, T] = {
+    SimpleFieldHolder(selector, initialValue, Nil, Nil, Nil)(valueConverter, valueSerializer)
+  }
   def field[T](selector: String)(implicit valueConverter: (String)=>Box[T], valueSerializer: (T)=>String): SimpleFieldHolder[T, T, T] = {
     SimpleFieldHolder[T,T,T](selector, Empty, Nil, Nil, Nil)(valueConverter, valueSerializer)
   }
 
+  def textareaField(selector: String, initialValue: String): TextareaHolder[String,String,String] = {
+    TextareaHolder[String,String,String](selector, Full(initialValue), Nil, Nil, Nil)
+  }
```

Comment:
The benefit of the above approach is we get to skip this function altogether… I like that because it means the question of what is a textarea or an input is up to the HTML, while the formality code is only concerned with the contents that come from and go to it.

If we do stick to this approach, `textareaField` has no reason to only work with strings. It should be parametrizable like `field` is.